packages/front-end/components/Features/AttributeModal.tsx

```diff
@@ -26,6 +26,16 @@ export interface Props {
   attribute?: string;
 }
 
+const DATA_TYPE_TO_DESCRIPTION: Record<SDKAttributeType, string> = {
```

Comment:
This is a small, unrelated UX improvement. Can pull this out if we don't want to include it in this PR